src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -242,7 +242,7 @@ private void VisitArray<T>(ImmutableArray<T> list, string header, bool logElemen
 
             var elementCount = logElementCount ? $"({list.Length})" : string.Empty;
             Indent();
-            LogString($"{header}{elementCount}: ");
+            LogString($"{header}{elementCount}:"+Environment.NewLine);
```

Comment:
This is the only tiny change. Rest of the changes are just adding a line to the output.